main.html

```diff
@@ -2,15 +2,32 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Week 1</title>
+    <title>Personal Portfolio</title>
     <link rel="stylesheet" href="main.css" />
   </head>
   <body>
     <header class="header">
       <nav class="nav">
-        <a href="#" class="logo">SJA</a>
-        <img class="btn" src="img/Union.png" alt="Union" />
+        <a href="#" class="logo">Project</a>
```

Comment:
- [ ] Hello. I suggest you replace "project" with your own name as shown in the original design